app/views/users/sessions/new.html.erb

```diff
@@ -46,7 +46,7 @@
           <div class="input-group">
             <%= f.submit "Log in" ,class:"btn primary-button users-login-primary-button" %>
             <div class="users-text-container">
-              <span class="users-text">New User?</span>
```

Comment:
"New User" seems fine; we don't have to explicitly mention CircuitVerse. Goes without saying

Reply:
Once,Please also check  the teacher's doc file(another change in this commit) .It would be better if I can change both of them in one go.Thanks!);